pkg/registration/hub/managedcluster/controller.go

```diff
@@ -126,8 +126,37 @@ func (c *managedClusterController) sync(ctx context.Context, syncCtx factory.Syn
 		// Hub cluster-admin denies the current spoke cluster, we remove its related resources and update its condition.
 		c.eventRecorder.Eventf("ManagedClusterDenied", "managed cluster %s is denied by hub cluster admin", managedClusterName)
 
-		if err := c.removeManagedClusterResources(ctx, managedClusterName); err != nil {
-			return err
+		// Apply(Update) the cluster specific rbac resources for this spoke cluster with hubAcceptsClient=false.
+		var errs []error
+		{
```

Comment:
what this bracket for?

Reply:
U-T added,
adding brackets can make `resourceResults` only affect inside it. It seems confused,  changed,